opsdroid/connector/facebook/__init__.py

```diff
@@ -34,9 +34,7 @@ def __init__(self, config, opsdroid=None):
         """Connector Setup."""
         super().__init__(config, opsdroid=opsdroid)
         _LOGGER.debug("Starting facebook connector")
-        self.config = config
```

Comment:
I assume this is now being dealt with elsewhere or isn't used any more?

Reply:
it's set in the base class.